x-pack/plugins/actions/server/integration_tests/__snapshots__/connector_types.test.ts.snap

```diff
@@ -2921,6 +2921,49 @@ Object {
             ],
             "type": "string",
           },
+          "headers": Object {
```

Comment:
Note that we added these checks to look for changes in connector params, which it noticed for `headers`.

In the future, we may require special handling when adding new optional parameters, to handle backward compatibility for serverless rollbacks.  For now, nothing actionable, but keep in mind if the relevant connector has an action queued in a newer version, which is then rolled back, the older code will likely fail validation for the new field.